src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -149,7 +157,9 @@ export default class AxisClipSliders extends React.Component<AxisClipSlidersProp
           </span>
           <span className="slider-name">{axis.toUpperCase()}</span>
           <span className="slider-slices">
-            {twoD ? `${sliderVals[0]} (${range.max})` : `${sliderVals[0]}, ${sliderVals[1]} (${range.max})`}
+            {twoD
+              ? `${sliderVals[0]} (${numSlices})`
+              : `${sliderVals[0]}, ${sliderVals[1]} (${sliderVals[1] - sliderVals[0] + 1})`}
```

Comment:
I'm slightly worried that this is now inconsistent for users, although hopefully understandable/interpretable in both cases.  
I would feel a tiny bit more comfortable if the 2d case used something like `index / max` instead of `index (max)` but we can run it by @lynwilhelm .. it's not a big deal for me.

Reply:
I ended up feeling tentatively comfortable that users would get it while using the feature, since they'll see that third number change in 3D mode, and understand that "number of slices selected" isn't meaningful in 2d mode. But that's maybe more discovery than we want to expect users to do themselves when we don't provide any cue of what the numbers mean, and in fact indicate that they might be the same thing. Maybe worth adding words as Lyndsay suggested to indicate the meaning of the numbers? Something like "13 (of 25)" or "12, 15 (3 slices)"? Definitely worth running by Lyndsay and the team regardless.

Relatedly, once merged, do we/should we want to leave the new changes deployed on nightly for a day or two to let the rest of the team experiment and offer feedback before marking a new version?

Reply:
Using nightly for smoke testing for a day or two makes sense to me, yes.

I still like the conciseness of the parentheses but maybe a hover tooltip over just the parenthetical would help explain it.  
On that note of user discoverability, did you do hover tooltips for the toolbar buttons?

Reply:
Yep - icon-only buttons on the toolbar have tooltips.